### PR TITLE
pfSense implementation under Xen - boot optimization

### PR DESCRIPTION
**Pull request by piotr-kleins**

### CODE REVIEW

**Comment by artur-rs** [comment]
@piotr-kleins your commits are not `Verified`

File: blog/content/post/2019-10-22-pfsense-boot-under-xen.md [hunk 1]
```diff
@@ -0,0 +1,350 @@
+---
+title: pfSense firewall boot process optimization under Xen hypervisor
+abstract: In previous article we introduce our implementation of pfSense under
+          Xen. Now, we want to show how you can improve boot process and reduce
+          virutalized pfSense boot time to minimum.
+cover: /covers/pfsense-logo.png
+author: piotr.kleinschmidt
+layout: post
+published: false
```
Comment (artur-rs) [inline blog/content/post/2019-10-22-pfsense-boot-under-xen.md]: `True`

File: blog/content/post/2019-10-22-pfsense-boot-under-xen.md [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+date: 2019-10-22
```
Comment (artur-rs) [inline blog/content/post/2019-10-22-pfsense-boot-under-xen.md]: `2019-12-04` and change the date in the filename

File: blog/content/post/2019-10-22-pfsense-boot-under-xen.md [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+categories:
+  - Firmware
+  - OS dev
+
+---
+
+## Introduction
+
+That article is continuation of [pfSense firewall implementation under Xen
+hypervisor](link). If you haven't read that yet, I strongly recommend to get
```
Comment (artur-rs) [inline blog/content/post/2019-10-22-pfsense-boot-under-xen.md]: no link?

**Comment by piotr-kleins** [comment]
@artur-rs I changed dates and GPG signed all commits

**Pull request merged**